package.json

```diff
@@ -29,83 +29,74 @@
     "build": "microbundle-crl --no-compress --format modern,cjs",
     "chromatic": "chromatic --project-token 60pcjug3hz",
     "lint": "eslint --ext ts,tsx src",
-    "prepare": "npm run build",
+    "prepare": "npm run build && husky install",
     "start": "microbundle-crl watch --no-compress --format modern,cjs",
     "storybook": "start-storybook -p 9009",
     "test": "jest",
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
     "types": "tsc --noEmit"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
-  "lint-staged": {
-    "src/**/*.{js,ts,jsx,tsx}": [
-      "eslint --fix"
-    ],
-    "*.json": [
-      "prettier --write"
-    ]
-  },
   "peerDependencies": {
-    "big.js": "^5.x",
-    "classnames": "^2.x",
+    "big.js": "^6.1.1",
     "react": "^16.x"
   },
-  "jest": {
-    "testEnvironment": "jsdom"
-  },
   "devDependencies": {
     "@babel/plugin-transform-runtime": "^7.15.0",
     "@babel/preset-env": "^7.11.0",
     "@babel/preset-react": "^7.10.4",
     "@babel/preset-typescript": "^7.10.4",
-    "@storybook/addon-docs": "^6.2.9",
-    "@storybook/addon-essentials": "^6.2.9",
+    "@storybook/addon-docs": "^6.4.21",
+    "@storybook/addon-essentials": "^6.4.21",
     "@storybook/addons": "^6.0.16",
-    "@storybook/preset-create-react-app": "^3.1.4",
     "@storybook/preset-typescript": "^3.0.0",
-    "@storybook/react": "^6.2.9",
+    "@storybook/react": "^6.1.21",
```

Comment:
wondering, why storybook is downgraded, approving anyways

Reply:
Hmm, good catch. I upgraded a bunch of Storybook libraries by running `npx sb upgrade`. It could be because we're using React 16 in this particular library right now.